### PR TITLE
align refs with HTTP style in h3 websockets draft

### PR DESCRIPTION
**Pull request by RyanTheOptimist**

### CODE REVIEW

**Comment by RyanTheOptimist** [comment]
@reschke I think I did the needful here by cargo-culting #2057. Does this look good to you?

File: draft-ietf-httpbis-h3-websockets.md [hunk 1]
```diff
@@ -32,9 +32,61 @@ author:
...
 
 normative:
-  HTTP/3: I-D.draft-ietf-quic-http-34
-  HTTP/2: I-D.draft-ietf-httpbis-http2bis-07
-  HTTP: I-D.draft-ietf-httpbis-semantics-19
+
+  HTTP:
+    display: HTTP
+    title: "HTTP Semantics"
+    date: 2022-03
```
Comment (reschke) [inline draft-ietf-httpbis-h3-websockets.md:39]:
```suggestion
    date: 2022-04
```

File: draft-ietf-httpbis-h3-websockets.md [hunk 2]
```diff
@@ -32,9 +32,61 @@ author:
...
+      -
+          ins: J. Reschke
+          name: Julian Reschke
+          org: greenbytes
+          role: editor
+
+  HTTP2:
+    display: HTTP/2
+    title: "HTTP/2"
+    date: 2022-03
```
Comment (reschke) [inline draft-ietf-httpbis-h3-websockets.md:63]:
```suggestion
    date: 2022-04
```

File: draft-ietf-httpbis-h3-websockets.md [hunk 3]
```diff
@@ -32,9 +32,61 @@ author:
...
+          role: editor
+      -
+          fullname: Cory Benfield
+          org: Apple Inc.
+          role: editor
+
+  HTTP3:
+    display: HTTP/3
+    title: "Hypertext Transfer Protocol Version 3 (HTTP/3)"
+    date: 2022-03
```
Comment (reschke) [inline draft-ietf-httpbis-h3-websockets.md:80]:
```suggestion
    date: 2022-04
```

**Review by reschke — APPROVED** [review]
(not tested, but if it ends generating the desired output...)

**Comment by RyanTheOptimist** [comment]
Thanks for the suggestions!

**Pull request merged**